docs_outdoornav_user_manual/integration_requirements/hardware_integration_requirements/hardware_integration_requirements.mdx

```diff
@@ -13,9 +13,9 @@ Select the checklist below that corresponds to your hardware kit.
 - [OutdoorNav Starter Kit Hardware Checklist](/docs_outdoornav_user_manual/integration_requirements/hardware_integration_requirements/starter_kit_hardware_checklist)
 - [OutdoorNav Custom Kit Hardware Checklist](/docs_outdoornav_user_manual/integration_requirements/hardware_integration_requirements/custom_kit_hardware_checklist)
 
-In addition, ensure that the platform computer is configured correctly by reviewing the following checklist.
+In addition, if a secondary/backpack/Starter Kit computer is being used, ensure that the platform computer is configured correctly by reviewing the following checklist.
```

Comment:
Do we need to define secondary and backpack? Or should we just change this to: 
"In addition, if the OutdoorNav software is not running on the platform computer itself, which is the case with the Starter Kit and some custom integrations, ensure that ..."